core/libraries/batch/JobHandlers/RegistrationsReport.php

```diff
@@ -347,7 +363,7 @@ public function get_csv_data_for($event_id, $offset, $limit, $question_labels, $
                     ARRAY_A,
                     'Payment_Method.PMD_admin_name as name, Payment.PAY_txn_id_chq_nmbr as gateway_txn_id, Payment.PAY_timestamp as payment_time'
                 );
-                list($payment_methods, $gateway_txn_ids_etc, $payment_times) = PaymentsInfoCSV::extractPaymentInfo($payments_info);
+                [$payment_methods, $gateway_txn_ids_etc, $payment_times] = PaymentsInfoCSV::extractPaymentInfo($payments_info);
```

Comment:
Is this correct? 

Now we have an array of arrays but we need each individual variable to be used later.

For example, Line 368 below uses the $payment_times var, but that's still set to an empty array from a little above this line.

Reply:
those are functionally equivalent.
The array on the assignment side of things is called "destructuring" and assigns each array element to a variable:

```php
 [ $a, $b, $c ] = [ 'A',  'B',  'C' ];
```

which works with the above code because `PaymentsInfoCSV::extractPaymentInfo()` returns an array.
 
My IDE does that automatically and I did not intend to make that change.

Reply:
>those are functionally equivalent.

Oh... cool! I did not know that. Thanks :)